onmetal-apiserver/internal/apis/storage/validation/volume_test.go

```diff
@@ -181,5 +191,20 @@ var _ = Describe("Volume", func() {
 			},
 			Not(ContainElement(ImmutableField("spec.volumePoolRef"))),
 		),
+		Entry("immutable encryption",
```

Comment:
There should be a test for changing the entire `encryption` field.

Reply:
Added testcases to Add/Remove entire `encryption` field